src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8297,6 +8297,40 @@ public void PathMapParser()
             Assert.Equal(1, parsedArgs.Errors.Count());
             Assert.Equal((int)ErrorCode.ERR_InvalidPathMap, parsedArgs.Errors[0].Code);
         }
+
+        [WorkItem(8470, "https://github.com/dotnet/roslyn/issues/8470")]
+        [Fact]
+        public void CompareSymbolDistinguisherDiagnosticArgumentsNoCompilation()
```

Comment:
Agree.  Can't we spin one of these up and directly call it with a `null` value for `Compilation`?  That would be a much more direct test.